components/Chips/examples/ChipsShapingExampleViewController.m

```diff
@@ -69,7 +56,10 @@ - (void)viewDidLoad {
   _chipView.accessoryPadding = UIEdgeInsetsMake(0, 0, 0, 10);
   CGSize chipSize = [_chipView sizeThatFits:self.view.bounds.size];
   _chipView.frame = CGRectMake(20, 20, chipSize.width + 20, chipSize.height + 20);
-  [_chipView applyThemeWithScheme:[self containerScheme]];
+  if (!self.containerScheme.colorScheme) {
+    self.containerScheme.colorScheme = [[MDCSemanticColorScheme alloc] init];
```

Comment:
Please use `MDCColorSchemeDefaultsMaterial201804` to be consistent with the default value in this (and most) examples.